pylint/extensions/typing.py

```diff
@@ -284,7 +307,20 @@ def leave_module(self, node: nodes.Module) -> None:
         'consider-using-alias' check. Make sure results are safe
         to recommend / collision free.
         """
-        if self._py37_plus and not self._py39_plus:
+        if self._py39_plus:
+            for msg in self._deprecated_typing_alias_msgs:
+                if (
+                    self._found_broken_callable_location
+                    and msg.qname == "typing.Callable"
+                ):
+                    continue
+                self.add_message(
+                    "deprecated-typing-alias",
+                    node=msg.node,
+                    args=(msg.qname, msg.alias),
```

Comment:
```suggestion
                    args=(msg.qname, msg.alias),
                    confidence=interfaces.INFERENCE,
```
AFAIK because of ``inferred.qname()`` line 275 / 282 ?

Reply:
Let's do that separately. It would change existing tests.